fe/fe-core/src/main/java/com/starrocks/external/iceberg/ExpressionConverter.java

```diff
@@ -0,0 +1,114 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+package com.starrocks.external.iceberg;
+
+import com.starrocks.analysis.BinaryPredicate;
+import com.starrocks.analysis.BoolLiteral;
+import com.starrocks.analysis.DateLiteral;
+import com.starrocks.analysis.DecimalLiteral;
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.FloatLiteral;
+import com.starrocks.analysis.IntLiteral;
+import com.starrocks.analysis.LiteralExpr;
+import com.starrocks.analysis.SlotRef;
+import com.starrocks.analysis.StringLiteral;
+import org.apache.iceberg.expressions.Expression.Operation;
+import org.apache.iceberg.expressions.Expressions;
+import org.apache.iceberg.expressions.UnboundPredicate;
+
+public class ExpressionConverter {
+
+    public static UnboundPredicate toIcebergExpression(Expr expr) {
+        if (! (expr instanceof BinaryPredicate)) {
+            return null;
+        }
+
+        BinaryPredicate predicate = (BinaryPredicate) expr;
+        Operation op = getIcebergOperator(predicate.getOp());
+        if (op == null) {
+            return null;
+        }
+
+        if (!(predicate.getChild(0) instanceof SlotRef)) {
+            return null;
+        }
+        SlotRef ref = (SlotRef) predicate.getChild(0);
+
+        if (!(predicate.getChild(1) instanceof LiteralExpr)) {
+            return null;
+        }
```

Comment:
Seems we currently only support expression like this pattern `var op literal` , actually there are more complex expression like `AND`, `OR`, `NOT` etc,  is there any plan to support in the next version ? 

In fact,  I don't suggest to add the filter push down in this PR ( Because we are implementing an incomplete filter pushdown in this PR). It's good to focus a feature in one PR.

Reply:
If plan to add filter push down for starrocks,  the iceberg's `SparkFilters` class is a good example to follow.

Reply:
As discussed with @imay we will submit an other pr to optimize this.